app/routes/task.js

```diff
@@ -0,0 +1,8 @@
+import Route from '@ember/routing/route';
+
+export default class TaskRoute extends Route {
+    model(params){
+        console.log(params.task_id)
```

Comment:
```suggestion
```

avoid committing console.logs and debugger to remote

can allow console.logs to be committed only in certain cases